pages/common/sha256sum.md

```diff
@@ -13,3 +13,7 @@
 - Read a file of SHA256 sums and verify all files have matching checksums:
 
 `sha256sum -c {{filename.sha256}}`
```

Comment:
Could we use the long argument format:
```suggestion
`sha256sum --check {{filename.sha256}}`
```

Reply:
The short argument was used in the previous line. Should I change that one too?

Reply:
Yeah, please.